domains/certificates/Query.API/API/appsettings.json

```diff
@@ -5,5 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+    "ConnectionStrings": {
+        "MartenDB": "PLACEHOLDER"
```

Comment:
I suggest that this conenction string is removed from appsettings.json. Appsettings.json is part of the configuration in all the ASPNETCORE_ENVIRONMENT (development, staging, production). When the application is running in a container, then it should read this from environment variables.